runtime/doc/filetype.txt

```diff
@@ -829,6 +829,13 @@ The mappings can be disabled with: >
 	let g:no_vim_maps = 1
 
 
+ZIG							*ft-zig-plugin*
+
+						*g:zig_recommended_style*
+The Zig |ftplugin| sets options according to Zig's recommended style. This can
+be disabled by setting |g:zig_recommended_style| to 0.
```

Comment:
Do we have a reference, what the Zig recommended style is?

Reply:
Can link to https://ziglang.org/documentation/0.12.0/#Style-Guide maybe? But that might link rot.

Reply:
Yes https://ziglang.org/documentation/master/#Whitespace
Should that be referenced in the section?

Reply:
Either like this, of have a look how it is done for python, explicitly mentioned what properties are set `:h ft-python-plugin`

Reply:
Fixed, I added both the link and the set fields